src/views/MainPage/components/OwnOrganization/Card/index.tsx

```diff
@@ -27,8 +27,21 @@ export default function OwnOrganizationCard({
   frontSideBg,
   backSideBg,
 }: OwnOrganizationCardProps) {
-  const isMobile = useIsMobile('1440px');
-  const backgroundSrc = isMobile ? frontSideBg.mo : frontSideBg.pc;
+  const isPC = useIsMobile('1440px');
+  const isTablet = useIsMobile('768px');
+  const isMobile = useIsMobile('376px');
+
+  const backgroundSrc = frontSideBg.bigPc
+    ? isMobile
+      ? frontSideBg.mo
+      : isTablet
+      ? frontSideBg.ta
+      : isPC
+      ? frontSideBg.pc
+      : frontSideBg.bigPc
+    : isPC
+    ? frontSideBg.mo
+    : frontSideBg.pc;
```

Comment:
이부분 뭔가 조건이 겹치는 부분도 있는 것 같은데 좀더 정리할 수 있지 않을까요?! 

bigPc 있으면 isMobile이면 mo, isTablet이면 ta, isPC면 pc, 셋다 아니면 bigPc 
bigPc 없다면 isPC면 mo, 아니면 pc  `-> 요거 맞나요?? pc면 pc, 아니면 mo 여야 하지 않나용 `

이거라면 
isMobile이면 mo, isTablet이면 ta, isPC면 pc, 셋다 아니면 bigPc (없으면 pc) 
이렇게 정리해도 되지 않나 싶은데 어떻게 생각하시나요!! 빠진 경우가 있을까요? 
만약 괜찮다면 아래처럼 정리할 수 있을 것 같아요 

```suggestion
  const backgroundSrc = isMobile
    ? frontSideBg.mo
    : isTablet
    ? frontSideBg.ta
    : isPC
    ? frontSideBg.pc
    : frontSideBg.bigPc || frontSideBg.pc;
```

Reply:
아 이게 왜 그랬냐면, 원래는 지금은 모두 다 이미지가 4개 있는데, 전에는 4개만 있는 것도 있고, 2개만 있는 것도 있어서 이에 따른 분기 처리를 따로 했었어서 그랬습니다!! 이 부분 수정하는 걸 깜빡했네요,, ㅎㅎ 감사합니다 ◡̈